eng/Get-TagsDocumentation.ps1

```diff
@@ -4,6 +4,7 @@ $ErrorActionPreference = 'Stop'
 $gitRepo = "https://github.com/microsoft/dotnet-framework-docker"
 
 & $PSScriptRoot/common/Invoke-ReadmeGeneration.ps1 dotnet/framework/runtime README.runtime.md manifest.json master $gitRepo
-& $PSScriptRoot/common/Invoke-ReadmeGeneration.ps1 dotnet/framework/sdk README.sdk.md manifest.json master $gitRepo -ReuseImageBuilderImage
+& $PSScriptRoot/common/Invoke-ReadmeGeneration.ps1 dotnet/framework/aspnet README.aspnet.md manifest.aspnet.json master $gitRepo
```

Comment:
Can you add the ` -ReuseImageBuilderImage` option to both the aspnet and wcf lines?  This will make it faster.